src/Uno.UI/UI/Xaml/Controls/ItemsControl/ItemsControl.skia.cs

```diff
@@ -16,11 +16,5 @@ public partial class ItemsControl : Control
 		{
 			InvalidateMeasure();
 		}
-
-		protected override Size MeasureOverride(Size availableSize)
-		{
-			UpdateItemsIfNeeded();
```

Comment:
That was not required?

Reply:
`UpdateItemsIfNeeded()` has been modified over time to only do something when invoked from `SetNeedsUpdateItems()`. Invoking this method by itself was not doing anything anymore, as there's no case where `_needsUpdateItems` is set without updating the items.

https://github.com/unoplatform/uno/blob/7f8d0c650def841284627313e3706415e78f0aef/src/Uno.UI/UI/Xaml/Controls/ItemsControl/ItemsControl.cs#L920-L936